core/templates/dev/head/header_css_libs.html

```diff
@@ -26,4 +7,6 @@
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
 <link rel="stylesheet" type="text/css" media="screen"
-      href="/third_party/static/font-awesome-4.4.0/css/font-awesome.min.css">
+      href="/css/oppia.css">
```

Comment:
I think the best order is to have all the fonts, then the third-party CSS, then oppia.css -- this will allow us to overwrite styles in third-party CSS where necessary.

Could you please check if changing the order here causes any errors (and fix any issues that occur)?